amq-streams/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Edit the CSV to disable FIPS_MODE
+  kubernetes.core.k8s:
+    state: patched
+    namespace: "{{ csv.metadata.namespace }}"
+    name: "{{ csv.metadata.name }}"
```

Comment:
same here. i get an error because i don't think we declare `csv` anywhere.

we can use `channel.currentCSV`

Reply:
I think I had tested this and didn't push the change to opcap-ansible. I'll rework with the existing info. It's all there.